docs/explanation/package-model.rst

```diff
@@ -1,9 +1,196 @@
+dependencies, a digital signature and many more fields.
+
+.. note::
+  The article :doc:`/reference/debian-dir-overview` showcases the layout of 
+  an unpacked *source package*.
+
+Source Package Formats
+~~~~~~~~~~~~~~~~~~~~~~
+
+There exist multiple formats for how the :term:`Source` is packaged. The format of a 
+*source package* is declared in the :file:`debian/source/format` file. This file should
+always exist. If this file can not be found, the :ref:`format 1.0 <SourcePackageFormat_1.0>`
+is assumed for backwards compatibility, but :manpage:`lintian(1)` will warn you about it.
+
+.. tip::
+
+    If you don't know what source format to use, you should probably pick either 
+    :ref:`3.0 (quilt) <SourcePackageFormat_3.0quilt>` or
+    :ref:`3.0 (native) <SourcePackageFormat_3.0native>`.
+
+
+.. _NativeSourcePackages:
+
+(Non-)Native Source Packages
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+In most cases, a software project is packaged by external contributors
+(:term:`Maintainers <Maintainer>`) only tangentially related to the software project. 
+Often, the *source package* has to do modifications to solve specific problems for its target
+:term:`Distribution`. The *source package* can, in these cases, be considered as its own
+software project, like a fork.
+
+Consequently, the :term:`Upstream` releases and *source package* releases do not always align.
+
+Native packages almost always originate from software projects designed with :term:`Debian`
+packaging in mind and have no independent existence outside its target :term:`Distribution`.
+Native packages do not differentiate between :term:`Upstream` releases and *source package* releases.
+Therefore, the version identifier of a native package does not have a 
+:term:`Ubuntu` or :term:`Debian` specific component.
+
+.. _SourcePackageFormat_3.0quilt:
+
+Format: ``3.0 (quilt)``
+^^^^^^^^^^^^^^^^^^^^^^^
+
+A new-generation *source package* format that records modifications in a :manpage:`quilt(1)`
+:term:`Patch` series within the :file:`debian/patches` folder. The :term:`Patches <Patch>` are
+organized as a :term:`Stack`, and you can apply, un-apply, and update them easily by traversing 
+the :term:`Stack` (push/pop). These changes are automatically applied during the extraction of
+the *source package*.
+
+A *source package* in this format contains at least an original tarball (``.orig.tar.ext`` where
+``ext`` can be ``gz``, ``bz2``, ``lzma`` and ``xz``) and a debian tarball (``.debian.tar.ext``).
+It can also contain additional original tarballs (``.orig-component.tar.ext``), where ``component``
+can only contain alphanumeric (``a-z``, ``A-Z``, ``0-9``) characters and hyphens (``-``).
+Optionally, each original tarball can be accompanied by a detached upstream signature
+(``.orig.tar.ext.asc`` and ``.orig-component.tar.ext.asc``).
+
+For example, take a look at the ``hello`` package:
+
+.. code:: bash
+
+    pull-lp-source --download-only 'hello' '2.10-3'
```

Comment:
This command doesn't work without first installing ubuntu-dev-tools

Reply:
After I installed, I got the following output:

Found hello 2.10-3 in mantic
Downloading hello_2.10-3.dsc from archive.ubuntu.com (0.002 MiB)
[=====================================================>]100%
Good signature by Santiago Vila <sanvila@debian.org> (0x41CE7F0B9F1B8B32)
Downloading hello_2.10.orig.tar.gz from archive.ubuntu.com (0.692 MiB)
[=====================================================>]100%
Downloading hello_2.10.orig.tar.gz.asc from archive.ubuntu.com (0.001 MiB)
[=====================================================>]100%
Downloading hello_2.10-3.debian.tar.xz from archive.ubuntu.com (0.012 MiB)
[=====================================================>]100%

Reply:
Installing `ubuntu-dev-tools` will be part of the getting set up tutorial. I add a note.

Reply:
I added a note to the first pull-lp-source to install ubuntu-dev-tools

Resolved with d95ce7b43232c1ae4f27b55c427d3369f63465e8